### PR TITLE
Change LDS renewal test and remove Typeform feedback form from Builder

### LINKED ISSUE #14708: Change Builder Feedback Form
Builder currently has a button to submit feedback:

image

It's pointing to the following typeform:

https://cartohq.typeform.com/to/t7Dk8K

We now have a new global feedback form here:

https://docs.google.com/forms/d/e/1FAIpQLScBQUWd-TP3Qy514DOCNg-KoLrViHijUR5giLAMS-3jmDnrPg/viewform

We should point the button to the new form.

### LINKED ISSUE #14724: [Dashboard] Unclear LDS Renewal Date
@micagm reported that the LDS Renewal Date is a bit confusing right now.

image

Text always says: "Your credits will be renewed on the Xth of *next* month" but they might be renewed this month (eg: in 5 days). 

If the credits will be renewed this month, we should say something like:

Your credits will be renewed on the Xth of *this* month.

cc: @kukukaka

### PR DESCRIPTION
**Pull request by jesusbotella**
This PR solves several issues:
- Unclear LDS Renewal Date. Closes #14724
- Change Builder Feedback Form. Closes #14708

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/components/Search/Suggestions/SearchSuggestions.vue [hunk 1]
```diff
@@ -3,9 +3,10 @@
     <ul v-if="searchResults" class="suggestions__content">
       <li :class="{'suggestions--active': activeSuggestionIndex === 0 }" @mouseover="updateActiveSuggestion(0)">
         <router-link
-          :to="{ name: searchRoute, params: searchRouteParameters }"
           class="suggestions__header is-caption text"
           :class="{ 'suggestions__header--loading': isFetching }"
+          :to="{ name: searchRoute, params: searchRouteParameters }"
```
Comment (csubira) [inline lib/assets/javascripts/new-dashboard/components/Search/Suggestions/SearchSuggestions.vue]: Is this change related to the form or to the renewal text?

Comment (jesusbotella) [reply lib/assets/javascripts/new-dashboard/components/Search/Suggestions/SearchSuggestions.vue]: Nope, sorry, that change is because I noticed that we weren't redirecting properly the users to the search from other static or RoR pages 😞

Comment (csubira) [reply lib/assets/javascripts/new-dashboard/components/Search/Suggestions/SearchSuggestions.vue]: No problem! In that case, this part and the rest LGTM!

**Comment by jesusbotella** [comment]
I've tested it myself and it works seamlessly 👍
